edb/common/debug.py

```diff
@@ -120,6 +120,9 @@ class flags(metaclass=FlagsMeta):
     pgserver = Flag(
         doc="Show PostgreSQL server logs and log all statements.")
 
+    telemetry = Flag(
```

Comment:
I wouldn't call this telemetry, just to avoid scaring people. Call it server_logging or something like that.